meta/Search.php

```diff
@@ -280,6 +280,8 @@ public function execute() {
         $this->result_pids = array();
         $result = array();
         $cursor = -1;
+        $pageidAndRevOnly = !(count($this->columns) > 2 || $this->columns[0]->getTid() ||
+            !empty($this->columns[1]) && $this->columns[1]->getTid());
```

Comment:
What if I select %pageid%, %title%, %lastupdate%? I think the more general way would be to check if all columns have a Tid of 0.